### ISSUE TITLE
Option for passing value not from stdin

### ISSUE THREAD

**Issue by ORESoftware** [stedolan/jq#1915]
I had this question on SO:
https://stackoverflow.com/a/56202936/5047085

I was looking for some cli option to pass JSON to jq not via stdin, something like:

```bash
jq -j '{"foo": 555}'  '.foo'
```

but that's not possible tmk, instead we can use this bash operator to read a string literal to stdin:

```bash
jq  '.foo' <<< '{"foo": 555}' 
```

but that's pretty fugly and probably slightly less performant - is there a way to pass strings to jq directly without another process?

**Comment by pkoppstein** [comment]
The responses dated May 18 by `oguz izmail` and by `peak` to [your SO question](https://stackoverflow.com/questions/56202911/jq-read-inline-string-not-from-stdin/56202936#56202936) provide several alternatives, e.g. using --argjson and environment variables.

**Comment by ORESoftware** [comment]
yeah `--argjson` is a long option name and the fact that I have to pass  `foo` and `bar` doesn't make sense to me, I am looking for something more concise and reasonable

**Comment by pkoppstein** [comment]
1. `--argjson` makes it easy to pass an entire JSON document. 

2. Positional arguments are supported with the `--args` option.

3. The previously mentioned responses provide other alternatives.

4. If you want to make a specific enhancement request, then you should either provide details or a PR, but be warned that the jq maintainers are loathe to add additional command-line options. In fact, based on past experience, I would say they will not do so if there is no added functionality.
